proposals/exception-handling/Exceptions.md

```diff
@@ -326,20 +381,20 @@ document](https://github.com/WebAssembly/spec/blob/master/document/core/text/ins
 The following rules are added to *instructions*:
 
 ```
-  try blocktype instruction* catch instruction* end |
+  try blocktype instruction* (catch instruction*)+ (catch_all instruction*)+ end |
```

Comment:
This grammar would require at least one catch and one catch_all, and allows multiple of the latter. I think you want
```suggestion
  try blocktype instruction* (catch instruction*)* (catch_all instruction*)? end |
```
This allows a try with no catch blocks, which I think is preferable anyway. If you want to rule that out, you'll need to complicate the grammar.

Reply:
I just want to add that issue #131 by @tlively also asks whether a try without further clauses could be allowed or not.